sample-app/screens/Home/Home.tsx

```diff
@@ -33,23 +34,84 @@ export default class Home extends Component {
 
   componentDidMount() {
     store.subscribe(() => {
-      this.getConfiguration()
+      this.getConfiguration(() => {
+        this.handleDeepLinkIfNeeded()
+      })
+    })
+
+    this.getConfiguration(() => {
+      this.handleDeepLinkIfNeeded()
     })
-    this.getConfiguration()
   }
 
   componentWillUnmount() {
     store.dispatch({ type: '' })
   }
 
-  async getConfiguration() {
+  async getConfiguration(callback: Function) {
     this.setState({ spinner: true })
     let configuration = await getStoredData(this.state)
     this.setState(configuration, () => {
-      this.setState({ spinner: false })
+      this.setState({ spinner: false }, () => {
+        callback()
+      })
     })
   }
 
+  async handleDeepLinkIfNeeded() {
+    const url = await Linking.getInitialURL()
+    if (url) {
+      this.updateDeepLinkURL(url, () => {
+        this.handlePBBATransaction()
+      })
+    }
+  }
+
+  updateDeepLinkURL(url: string | null, callback: Function) {
+    const mockURL = url + '&orderId=BTO-cUOLVpSbQ1Ce9M3a3Mc7_w'
```

Comment:
is this still needed?

Reply:
@stpopa There's still no **orderId** in the URL.